docs/source/guide/zne-2-use-case.myst

```diff
@@ -0,0 +1,43 @@
+---
+jupytext:
+  text_representation:
+    extension: .myst
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.10.3
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# When should I use ZNE?
+ 
+## Advantages 
+
+ZNE generally works well for circuits of low depth and/or those exhibiting noise 
+that can be described by a simple model 
+[[32](https://mitiq.readthedocs.io/en/latest/zz_bibliography.html)]. 
```

Comment:
```suggestion
Zero noise extrapolation is one of the simplest error mitigation techniques and, in many practical situations, it can be applied with a relatively small sampling cost.
```
I would suggest this change since I think that "simplicity" and "low-cost" are the main advantages of ZNE.  The fact that it works better for low-depth circuits is true but it is more a disadvantage than an advantage :-)